lib/Converter/FunctionConverter.cpp

```diff
@@ -126,6 +126,12 @@ void FunctionConverter::convertOutputs(Node &node) {
       if (user == conversionVal.getNode()) {
         continue;
       }
+      // Log the change of node input(operand).
+      if (node.getParent()) {
+        node.getParent()->getLogContext().logNodeInputChange(user, *(use.get()),
+                                                             conversionVal);
+      }
+
       use.get()->setOperand(conversionVal.getNode(), conversionVal.getResNo());
     }
```

Comment:
I think it make sense to also `logNodeReplacement()` here too, WDYT?

This function loops over all of the `NodeValues` from the input `node` and replaces each one with `conversionVal`. So this is essentially a specialized `replaceAllUsesOfWith()`.

Reply:
Yea, that's right. However i found that in this scope(convertOutputs()) the conversionVal actually may belong to different function, which means it's one nodeVal replaced by several nodeVal for different users.

Reply:
Actually call nodeReplacement for each different conversionVal may produce some messy output that makes parsing more difficult